clang/test/C/C23/n3029.c

```diff
@@ -0,0 +1,59 @@
+// RUN: %clang_cc1 -verify -triple x86_64-unknown-linux-gnu -fsyntax-only --embed-dir=%S/Inputs -std=c23 %s -pedantic -Wpre-c23-compat
```

Comment:
```suggestion
// RUN: %clang_cc1 -verify -triple x86_64-unknown-linux-gnu -fsyntax-only -std=c23 %s -pedantic -Wpre-c23-compat
```

Reply:
You should split this into two `RUN` lines, one `-std=c23 -Wpre-c23-compat` and the other `-std=c17 -pedantic` so that we see the pedantic warnings being tested.

Reply:
Done.